src/components/containers/headers/__test__/headers.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { shallow } from "enzyme";
+import HeaderComponent from "../index";
+
+describe("General HeaderComponent :", () => {
```

Comment:
this file doesn't test anything, we would rather have it removed.

Reply:
I have decided to add more tests. Please wait for the update.